apps/src/applab/designMode.js

```diff
@@ -469,6 +502,12 @@ designMode.serializeToLevelHtml = function () {
     elementUtils.removeIdPrefix(this);
   });
 
+  // Remove the "data:img/png..." URI from icon images
+  designModeVizClone.find('[data-canonical-image-url^="icon:"]').each(function () {
+    this.removeAttribute('src');
+    this.style.backgroundImage = '';
+  });
```

Comment:
The data-URI of the rendered glyph adds a long string to the DOM.  We don't want to save this data in `levelHtml`, so I remove it here.  When the user loads the project, the `src` and `background-image` properties get regenerated from `data-canonical-image-url`.